### DISCUSSION TITLE
Sorting Stats by Layout

### DISCUSSION THREAD

**Discussion by sadiemay711** [monkeytypegame/monkeytype#972] [Ideas] [answered]
I was wanting a feature to sort stats about tests by keyboard layout, and possibly tagging older tests by layout. I'm learning Dvorak and right now all of my tests are grouped together with no way of sorting them. I just thought it'd be useful to get a more accurate grasp on improvements. Have a nice day!

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
Hey I don't know if this helps but a way you can separate your typing on different layouts would be using the tag feature. If you have a tag on while you're typing then all your results will be on that specific tag. So as an example you could be using the Dvorak keyboard layout while using the tag name Dvorak and a QWERTY keyboard layout on a tag named QWERTY. This would let you accurately grasp your improvements.

To create tags you go up to the gear at the top and click it, then you will see something that says account and under that you can make tags and use them. You can also use tags through the command line (hit escape to use it) but you can't make tags through it.

**Reply by sadiemay711** [reply]
Heck yeah, thank you! That'll work great for what I'm trying to do with it.

**Reply by frogfrogfrogfrogfrog** [reply]
:D
